test/common/http/utility_test.cc

```diff
@@ -471,7 +472,30 @@ TEST(HttpUtility, SendLocalGrpcReply) {
         EXPECT_NE(headers.GrpcMessage(), nullptr);
         EXPECT_STREQ(headers.GrpcMessage()->value().c_str(), "large");
       }));
-  Utility::sendLocalReply(true, callbacks, is_reset, Http::Code::PayloadTooLarge, "large", false);
+  Utility::sendLocalReply(true, callbacks, is_reset, Http::Code::PayloadTooLarge, "large",
+                          absl::nullopt, false);
+}
+
+TEST(HttpUtility, RateLimitedGrpcStatus) {
+  MockStreamDecoderFilterCallbacks callbacks;
+
+  EXPECT_CALL(callbacks, encodeHeaders_(_, true))
+      .WillOnce(Invoke([&](const HeaderMap& headers, bool) -> void {
+        EXPECT_NE(headers.GrpcStatus(), nullptr);
+        EXPECT_STREQ(headers.GrpcStatus()->value().c_str(), "14"); // Unavailable
```

Comment:
hmm interesting, we dont have a utility map for this?

Reply:
Addressed via 104bc64.